lasy/profiles/transverse/hermite_gaussian_profile.py

```diff
@@ -5,13 +5,12 @@
 
 
 class HermiteGaussianTransverseProfile(TransverseProfile):
-    """
-    Derived class for an analytic profile of a high-order Gaussian
-    laser pulse expressed in the Hermite-Gaussian formalism.
+    r"""
+    A high-order Gaussian laser pulse expressed in the Hermite-Gaussian formalism.
 
-    More precisely, the transverse envelope
-    (to be used in the :class:CombinedLongitudinalTransverseLaser class)
-    corresponds to:
+    Derived class for an analytic profile.
+    More precisely, the transverse envelope (to be used in the
+    :class:CombinedLongitudinalTransverseLaser class) corresponds to:
```

Comment:
While in principle it is nice that it can automatically rephrase docstrings, I wonder if it might be a bit too intrusive sometimes (and make the text less clear). I would probably prefer to disable this check, if possible.

Reply:
Oh, that was me to be honest.

The only thing it rephrases grammar-wise is it warns if imperative is not used.